Week_12/Ejercicio_2.py

```diff
@@ -0,0 +1,50 @@
+from abc import ABC, abstractmethod
+
+class Shape(ABC):
+
+    @abstractmethod
+    def area(self):
```

Comment:
Functions and methods should always use verbs to describe the action they do.
In this case, area dn perimeter just return such values, so we should call them '**get**_area' and '**get**_perimeter'

Next week we see decorators, and one of them is the property. If these were properties, then the names would be good

Reply:
Changed to follow naming conventions:
![image](https://github.com/JorgeA/DUAD/assets/4186222/2043e14b-27fe-4e8e-b406-3416aeb7cca7)

Reply:
renamed to follow naming conventions:
![image](https://github.com/JorgeA/DUAD/assets/4186222/ce643a70-c465-42dd-b882-7342bc3ec98a)